src/sqerl_client.erl

```diff
@@ -119,22 +119,10 @@ start_link(DbType) ->
     gen_server:start_link(?MODULE, [DbType], []).
 
 init([]) ->
-    init(drivermod());
-init(CallbackMod) ->
-    IdleCheck = envy:get(sqerl,idle_check, 1000, non_neg_integer),
+    init(drivermod(), config()).
 
-    Statements = read_statements_from_config(),
-
-    %% The ip_mode key in the sqerl clause determines if we parse db_host as IPv4 or IPv6
-    Config = [{host, envy_parse:host_to_ip(sqerl, db_host)},
-              {port, envy:get(sqerl, db_port, pos_integer)},
-              {user, envy:get(sqerl, db_user, string)},
-              {pass, envy:get(sqerl, db_pass, string)},
-              {db, envy:get(sqerl, db_name, string)},
-              {timeout, envy:get(sqerl,db_timeout, 5000, pos_integer)},
-              {idle_check, IdleCheck},
-              {prepared_statements, Statements},
-              {column_transforms, envy:get(sqerl, column_transforms, list)}],
+init(CallbackMod, Config) ->
```

Comment:
Reading the PR I got confused because the title of the change is "callback mod" but this is the driver module, not the config one. I realize not related to this change, but wonder if a variable name change to `DriverMod` would be a readability improvement.